### PR TITLE
Cameronperera/app token delete

### PR DESCRIPTION
**Pull request by cameronperera**

### CODE REVIEW

File: internal/apptoken/repository_delete_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			reader:          testRw,
+			writer:          testRw,
+			kms:             testKms,
+			publicId:        testOrg.PublicId,
+			createdBy:       testUserHistoryId,
+			grantsStr:       "id=*;type=*;actions=*",
+			wantRowsDeleted: 1,
+		},
+	}
+	for _, tc := range tests {
```
Comment (jimlambrt) [inline internal/apptoken/repository_delete_test.go:55]: Looks like you're missing some of the test code here.

**Review by jimlambrt — CHANGES_REQUESTED** [review]

File: internal/apptoken/repository_delete_test.go (hunk 1 above)
Comment (cameronperera) [reply internal/apptoken/repository_delete_test.go:61]: Ope. Thanks for catching that. I staged my changes but didn't actually commit them.

**Review by jimlambrt — APPROVED** [review]
Ty!

**Review by uruemu — APPROVED** [review]
LGTM

**Pull request merged**